src/qt/assetcontroldialog.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2011-2015 The BitAsset Core developers
+// Copyright (c) 2011-2015 The Bitcoin Core developers
```

Comment:
we need to add raptoreum copyright here

Reply:
this file already has the Raptoreum copyright headers